ios/brave-ios/Sources/BraveWallet/Preview Content/MockContent.swift

```diff
@@ -154,6 +154,24 @@ extension BraveWallet.BlockchainToken {
     chainId: BraveWallet.FilecoinMainnet,
     coin: .fil
   )
+
+  static let mockBTCToken: BraveWallet.BlockchainToken = .init(
+    contractAddress: "",
+    name: "Bitcoin",
+    logo: "",
+    isErc20: false,
```

Comment:
nit: we should probably be making BlockchainToken an interface specific to the coin type rather than making these overlap between all of the networks